### DISCUSSION TITLE
Spherical map rendering

### DISCUSSION THREAD

**Discussion by lacrossj** [Stellarium/stellarium#551] [Q&A] [answered]
I would love to be able to render a spherical map. Something I can map to a sphere in a in a 3D software like Max or Maya. Ideally It could render at very large resolutions. 
Currently I can try to build them from a bunch of screen shots but the screen shots are limited to the monitor resolution and I can't seem to type in percises viewer camera rotations.

**Comment by gzotti** [comment]
You can use scripting functions to point to any location, set projection type and FoV. Set screenshot size to what you want, and make a screenshot. And read the Guide for more details.

**Comment by lacrossj** [comment]
That's great, but that is target centric. It would also be great to have something camera centric. So that I could rotate the camera 10 degrees on its Y axis for example. I realized that this software is coming from a different perspective than I am. I'm visual effects artist. I often have to put stars in skies. Thos skies need to be complete domes. Most clients don't care about accuracy but if I want to be accurate, there are very few options.

**Comment by gzotti** [comment]
If you target coordinates, you can do almost everything. Just move in azimuth and altitude. You cannot rotate the camera in a rolling fashion (along the optical axis), though.

**Comment by lacrossj** [comment]
Thanks for your help. and sorry for being a "noob" but can you explain to me how I can do that? I've been looking through all the gui and can't seem to find anything. I've only been able to rotate the camera by mashing the left and right arrow keys. Does that require some scripting?

**Comment by gzotti** [comment]
My first answer was to use scripting functions, yes. https://stellarium.org/doc/0.18/scripting.html

Find StelMovementMgr::moveToJ2000(), moveToAltAzi(), zoomTo().

**Comment by lacrossj** [comment]
Cool. I was just a little surprised there wasn't a gui way of doing that but I hear ya. Thanks for your help.

**Comment by gzotti** [comment, marked as answer]
Regular users don't need it, they operate much more intuitively with the mouse. Such GUI is in the specialized RemoteControl plugin web GUI. 
Use http://localhost:8090/tablet7in.html
